app/javascript/mastodon/components/alt_text_badge.tsx

```diff
@@ -47,9 +54,12 @@ export const AltTextBadge: React.FC<{
       >
         {({ props }) => (
           <div {...props} className='hover-card-controller'>
-            <div
+            <div // eslint-disable-line jsx-a11y/no-noninteractive-element-interactions
               className='media-gallery__alt__popover dropdown-animation'
-              role='tooltip'
+              role='region'
```

Comment:
I'm no expert, but https://inclusive-components.design/tooltips-toggletips/#toggletipswithliveregions seems to suggest `role="status"`